<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1467</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>20</number>
    </property>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignTop">
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top: 50px; margin-left: 10px</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../../../Downloads/tes.qrc">
          <normaloff>:/newPrefix/Back.png</normaloff>:/newPrefix/Back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>47</width>
          <height>49</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(28, 29, 34)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>25</x>
           <y>50</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../../../../../../../Downloads/tes.qrc">:/newPrefix/Items.png</pixmap>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>31</x>
           <y>190</y>
           <width>31</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/todolist.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>21</x>
           <y>270</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/daily plan.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>22</x>
           <y>350</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/target.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>21</x>
           <y>440</y>
           <width>51</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/gratitude.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>510</y>
           <width>41</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/article.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>730</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/Icon.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>36</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top: 40px; color : #2F2F2F;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Daily Plan&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="styleSheet">
       <string notr="true">
#calendarWidget QWidget {
	border-radius: 80px;
	font-size: 16px;
	font-weight: bold;
}

#qt_calendar_navigationbar {
	border: 2px solid #00C2C3;
	background-color: #00C2C3;
	color: white;
	border-radius: 20px;
	border-bottom: 0px;
}

#qt_calendar_yearbutton {
	color: #fff;
	margin: 5px;
	font-size: 24px;
	padding: 5px 10px;
	font-weight: bold;
}
#qt_calendar_nextmonth{
  
  min-width:20px;
  max-width:20px;
  min-height:20px;
  max-height:20px;
  
  border-radius:5px;
  background-color:transparent;
  padding:5px;
}
#qt_calendar_prevmonth::hover{
  qproperty-icon:url(&quot;../prev_icon.png&quot;);
  min-width:10px;
  max-width:10px;
  min-height:10px;
  max-height:10px;
}

#qt_calendar_monthbutton {
	color: #fff;
	margin: 5px;
	font-size: 24px;
	padding: 5px 10px;
	font-weight: bold;
}

QCalendarWidget QToolButton#qt_calendar_nextmonth::right-arrow {icon: url(./images/next_icon.png);}</string>
      </property>
      <property name="selectedDate">
       <date>
        <year>2023</year>
        <month>4</month>
        <day>12</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2023</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="firstDayOfWeek">
       <enum>Qt::Monday</enum>
      </property>
      <property name="gridVisible">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QCalendarWidget::SingleSelection</enum>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::NoVerticalHeader</enum>
      </property>
      <property name="dateEditEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../../../../Downloads/tes.qrc"/>
  <include location="../../../../../../../../Downloads/tes.qrc"/>
  <include location="../../../../../../../../Downloads/tes.qrc"/>
 </resources>
 <connections/>
</ui>
